.github/workflows/fabric_acctest.yml

```diff
@@ -44,6 +46,79 @@ jobs:
     steps:
       - run: true
 
+  configure-oidc-template:
+    name: Configure OIDC Subject Claim Template
+    needs: authorize
+    runs-on: ubuntu-latest
+    steps:
+      - name: Configure OIDC Subject Claim Template
```

Comment:
Is this something that only needs to be done once per repo, or does the template get reset periodically?  If it's a one-time action and doesn't have to be executed every time an OIDC token is requested, we should document the requirement somewhere but not include this step--or the `actions: write` permission--in the workflow.